pkg/reconciler/events/cloudevent/cloudeventclient.go

```diff
@@ -59,7 +62,38 @@ func withCloudEventClient(ctx context.Context) context.Context {
 		logger.Panicf("Error creating the cloudevents client: %s", err)
 	}
 
-	return context.WithValue(ctx, ceKey{}, cloudEventClient)
+	celient := CloudClient{
+		client: cloudEventClient,
+	}
+	return context.WithValue(ctx, ceKey{}, celient)
+}
+
+// CloudClient is wrapper of cloud client and implements AddCount and DecreaseCount
```

Comment:
NIT: CloudClient is a wrapper of CloudEvents client and implements addCount and decreaseCount

Reply:
Thank you!